apps/judicial-system/web/src/routes/PublicProsecutor/Indictments/Overview/IndictmentOverview.tsx

```diff
@@ -0,0 +1,187 @@
+      )
+      .map(({ id, name }) => ({
+        label: name ?? '',
+        value: id,
+      }))
+  }, [data?.users, user])
+
+  return (
+    <PageLayout
+      workingCase={workingCase}
+      isLoading={isLoadingWorkingCase}
+      notFound={caseNotFound}
+      isValid={true}
+      onNavigationTo={handleNavigationTo}
+    >
+      <PageHeader
+        title={fm(titles.shared.closedCaseOverview, {
+          courtCaseNumber: workingCase.courtCaseNumber,
+        })}
+      />
+      <FormContentContainer>
+        <PageTitle>{fm(strings.title)}</PageTitle>
+        <CourtCaseInfo workingCase={workingCase} />
+        <Box component="section" marginBottom={5}>
+          <InfoCardClosedIndictment />
+        </Box>
+        {lawsBroken.size > 0 && (
+          <Box marginBottom={5}>
+            <IndictmentsLawsBrokenAccordionItem workingCase={workingCase} />
+          </Box>
+        )}
+        {workingCase.caseFiles && (
+          <Box component="section" marginBottom={5}>
+            <IndictmentCaseFilesList workingCase={workingCase} />
+          </Box>
+        )}
+        <Box marginBottom={5}>
+          <SectionHeading
+            title={fm(strings.reviewerTitle)}
+            description={
+              <Text variant="eyebrow">
+                {fm(strings.reviewerSubtitle, {
+                  indictmentAppealDeadline: formatDate(
+                    workingCase.indictmentAppealDeadline,
+                    'P',
+                  ),
+                })}
+              </Text>
+            }
+          />
+          <BlueBox>
+            <Select
+              name="reviewer"
+              label={fm(strings.reviewerLabel)}
+              placeholder={fm(strings.reviewerPlaceholder)}
+              value={selectedIndictmentReviewer}
+              options={publicProsecutors}
+              onChange={(value) => {
+                setSelectedIndictmentReviewer(value as Option<string>)
+              }}
+              isDisabled={loading}
+              required
+            ></Select>
+          </BlueBox>
+        </Box>
+      </FormContentContainer>
+
+      <FormContentContainer isFooter>
+        <FormFooter
+          nextButtonIcon="arrowForward"
+          previousUrl={`${constants.CASES_ROUTE}`}
+          nextIsLoading={isLoadingWorkingCase}
+          nextIsDisabled={!selectedIndictmentReviewer || isLoadingWorkingCase}
+          onNextButtonClick={assignReviewer}
+          nextButtonText={fm(core.continue)}
+        />
+      </FormContentContainer>
+
+      {modalVisible && (
+        <Modal
+          title={fm(strings.reviewerAssignedModalTitle)}
+          text={fm(strings.reviewerAssignedModalText, {
+            caseNumber: workingCase.courtCaseNumber,
+            reviewer: selectedIndictmentReviewer?.label,
+          })}
+          secondaryButtonText={fm(core.back)}
+          onSecondaryButtonClick={() => router.push(constants.CASES_ROUTE)}
+        ></Modal>
```

Comment:
I think it's a cleaner syntax to write `/>` instead of `></Modal>`. Please consider replacing ☺️

Reply:
True! Sloppy, fixed 😊